gradle.properties

```diff
@@ -21,34 +21,32 @@ org.gradle.jvmargs=-Xms1g
 
 #3rd party Versions
 apacheCommonsCsvVersion=1.5
-apacheCommonsCompressVersion=1.20
+apacheCommonsCompressVersion=1.21
```

Comment:
can we have separate PRs for Java dependencies upgrades and the Docker image changes ?

Reply:
We are testing them together by running security scans on the final image so having them in a single branch makes the testing easier.  We may consider splitting them once all the testing is complete

Reply:
I understand